scripts/lb-wrapper

```diff
@@ -6,6 +6,7 @@ URL_OR_SEARCH_TERM="$2"
 LOG_FILE="/var/log/xklb.log"
 XKLB_DB_FILE="/library/calibre-web/xklb-metadata.db"
 TMP_DOWNLOADS_DIR="/library/downloads/calibre-web"
+OUTTMPL="'${TMP_DOWNLOADS_DIR}/%(extractor_key,extractor)s/%(uploader,uploader_id)s/%(title).170B_%(view_count)3.2D_[%(id).64B].%(ext)s'"
```

Comment:
```suggestion
# outtmpl explanation:
# https://github.com/yt-dlp/yt-dlp#output-template
# https://github.com/chapmanjacobd/library/blob/e682ac20f5dcc4fa94238b57931f4705b9515059/xklb/createdb/tube_backend.py#L385-L388
OUTTMPL="'${TMP_DOWNLOADS_DIR}/%(extractor_key,extractor)s/%(uploader,uploader_id)s/%(title).170B_%(view_count)3.2D_[%(id).64B].%(ext)s'"
```

Reply:
Did you see the edit I made in line numbers here? Committed in https://github.com/iiab/calibre-web/pull/177/commits/397b8c2283547fbdc857efa74b44d4fd08a4e9d0

Reply:
Yep, thanks.

Request is both code snippets if possible: https://github.com/iiab/calibre-web/pull/177#discussion_r1633224654